packages/web/src/scss/components/Pagination/_Pagination.scss

```diff
@@ -0,0 +1,44 @@
+@use 'theme';
+@use '../../settings/cursors';
+
+.Pagination {
```

Comment:
We agreed on styling only the `item` part of the component BUT in BEM methodology it is an element, actually a middle element from BEM letters that stands for its meaning.

Reasons why `Pagination` block was styled:

- There is no utility/helper class to set gaps between items;
- and I wasn't really into using `margin-inline` for each item;
- and it is handy to set `margin-inline: auto;` to Pagination block so in case a usage on project requires pagination horizontal align to center, it's simple by just adding `d-flex` to its parent.